tests/autoyast/repos.pm

```diff
@@ -25,6 +25,8 @@ sub run {
     # sles12_minimal.xml profile does not install "ip"
     assert_script_run 'ip a || ifstatus all';
     pkcon_quit;
+    # If no repository is enabled, enable all repositories
+    script_run('zypper lr |cut -d \| -f 4 |grep " Yes " 2>&1>/dev/null') && zypper_call 'mr -ea';
```

Comment:
The only problem I see with this command is that it will break once we have `Enabled` as different column in the output. Good part about #6796 that it also limits it to the case when it's actually required (which is only all packaged DVD case).

Reply:
It won't match "Enabled" since we grep " Yes ", or I misunderstand what you mean. I am testing a mix of both : check if we are in a case where this is required as in  #6796, then check if the cd/dvd repo is disabled, and only then, enable local repos with zypper mr -el (not all amymore) just in case things would change in the future.

Reply:
I mean here you rely that "enabled" if 4-th column, if happens that it becomes 3-rd or 5-th, this will break, so I would use settings from the job as in #6796 to trigger this behavior.

Reply:
I don't see why the order of columns would change. I tested this in various situations. 
The thing is, #6796 is fine but if there is a change again, eg if in 15.2 the repos are enabled, then it will try to enable the repos even if they are already enabled. 
But anyway, it does not really matter : 
`susetest:/home/bernhard # zypper mr -el`
`Nothing to change for repository 'SLES15-SP1-15.1-0'.`
`susetest:/home/bernhard # echo $?`
`0`
"zypper mr -el" returns 0 in any case, so we could as well not test anything.

Reply:
We already get that changed for snapper, so it's hard to predict. The only thing I would like to see is that we duplicate #6796, so could you please extract yours/other implementation and use it in both places?